services/hockeyapp/hockeyapp.tester.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const createServiceTester = require('../create-service-tester')
+
+const { isVPlusDottedVersionAtLeastOne } = require('../test-validators')
+
+const t = createServiceTester()
+module.exports = t
+
+t.create('Invalid AppToken or AppId')
+  .get('hockeyapp/v/1234/1234.json')
+  .expectJSONTypes({ name: 'hockeyapp', value: 'invalid' })
+
```

Comment:
We should have at least one valid case test in the suite which hits the live API and doesn't mock the response. I realise it is unusual practice for unit tests to depend on an external service ("don't test what you don't own") but we integrate with hundreds of services. If an upstream API changes or goes away, we need a test to start failing on the daily build otherwise we won't notice it.

Reply:
Hey I totally understand that but we would have to rely on an app token that has to be obtained with a real account at https://hockeyapp.net - how do you handle this situation?